lib/id3c/utils.py

```diff
@@ -94,3 +94,64 @@ def shorten(text, length, placeholder):
         return text[0:length - len(placeholder)] + placeholder
     else:
         return text
+
+
+def contextualize_char(text, idx, context = 10):
```

Comment:
Just commenting, I'll be curious to see if using the default here (up to) 10 characters on either side ends up being enough to debug these. I think trying to limit leakage of potential phi makes sense though, so this seems like a good starting point.

Reply:
Yeah, we'll see! I think 10 chars of surrounding context will be sufficient for many actual JSON encoding errors, but I don't really expect actual encoding errors that often.

More often I expect that the response body isn't JSON at all (even if it is supposed to be) and that this will result in an error at position 0, which triggers the first branch of this conditional:

https://github.com/seattleflu/id3c/blob/74611ac5c11fd1be93ab60ce276b4cf4395f23d1/lib/id3c/json.py#L96-L100